src/test/test_all.ts

```diff
@@ -6,6 +6,13 @@ let exitCode = 0;
 const cwd = process.cwd();
 const testEnv = Object.create(process.env) as NodeJS.Dict<string>;
 
+// Read command line arguments for test filtering
+const testFilterArgs = process.argv.slice(2);
+if (testFilterArgs.length > 0) {
+	testEnv.DART_CODE_TEST_FILTER = testFilterArgs.join(" ");
```

Comment:
We need to support passing multiple files, for example:

```
npm test myfile1 my file2
```

And this should run both `test/myfile1.test.js` and `test/myfile2.test.js`. I don't think joining them here and then just putting them in the glob will work.